### DISCUSSION TITLE
How to monitor for changes in ente's docker image?

### DISCUSSION THREAD

**Discussion by roughnecks** [ente-io/ente#1703] [Q&A]
I mean: ghcr.io/ente-io/server

Thanks

**Comment by mnvr** [comment]
Each image is tagged by the commit ID

<img width="684" alt="Screenshot 2024-05-12 at 21 17 10" src="https://github.com/ente-io/ente/assets/24503581/b1fd2c2c-5a97-408c-92f7-e95218b2cc65">

Also, there is a GitHub tag, [`server/ghcr`](https://github.com/ente-io/ente/tree/server/ghcr), that will always point to the commit of the latest image.

If your question is if there is a way to be notified whenever a new image is published, hmm, not sure. I don't know if GitHub allows subscribing to updates to its container registry, but people might have found workarounds. If nothing else works, you could poll the ghcr listing for the image or the github tag and detect updates. I'd recommend searching around first, this seems like a common wish and other folks might've found solutions for watching docker registries for updates.
